sync/work/build.gradle.kts

```diff
@@ -33,7 +33,6 @@ dependencies {
     implementation(project(":core:datastore"))
     implementation(project(":core:model"))
     implementation(libs.androidx.lifecycle.livedata.ktx)
-    implementation(libs.androidx.startup)
```

Comment:
This was our only usage of `androidx.startup`, so it might be worth removing entirely from `libs.versions.toml`?